### DISCUSSION TITLE
ios action, possibile raggrupparle?

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2438] [GENERALE - Discussioni generiche]
Ciao a tutti, ho creato vari pulsanti da watch tramite ios: action: regolarmente funzionanti e dichiarte su configuration.yaml.
e' possibile raggrupparli in un menù che a sua volta apre i vari pulsanti?
esempio
pulsante Aperture all'interno action per cancello, garage, tende etc etc
pulsante luci, le varie scene/automazioni delle luci
e via cosi.

**Comment by Scialla** [comment]
non mi risulta sia possibile e non lo vedo nella documentazione ufficiale
https://companion.home-assistant.io/docs/getting_started/

Sinceramente, e personalmente, vedo le shortcut su AW utili per comandi ripetitivi e molto utilizzati. Pensare di avere un menù da "sfogliare"e comandare tutto da AW lo ritengo assurdo ma, ribadisco, è solo una mia opinione; meglio farlo con siri, molto più veloce
